Makefile.am

```diff
@@ -456,5 +462,5 @@ MAINTAINERCLEANFILES = $(BUILT_SOURCES)
 .a65.bin:
 	o65file=`echo $@ | sed 's/bin/o65/'`;\
 	xa -R -G $< -o $$o65file &&\
-	od -v -An -w8 -tx1 $$o65file | sed -re 's/[[:alnum:]]+/0x&,/g' > $@ &&\
+	${OD_BIN} -v -An -w8 -tx1 $$o65file | sed -re 's/[[:alnum:]]+/0x&,/g' > $@ &&\
```

Comment:
I'd rather remove the _BIN suffix, would be more conformant to the style of other command variables such as ${RM}

Reply:
Done!